oemoflex/component_attrs/pv.csv

```diff
@@ -1,12 +1,12 @@
 attribute,type,unit,default,suffix,description
 region,str,n/a,n/a,,Country code
-name,str,n/a,,-el-solarpv,Name of the bus
+name,str,n/a,,-el-solarpv,Name
 type,str,n/a,volatile,,Type
-carrier,str,n/a,solar,,
-tech,str,n/a,pv,,
-bus,str,n/a,,-el-bus,
-capacity,,,,,
-capacity_cost,,,,,
-marginal_cost,float,Eur/kWh,0,,
-profile,,,,-el-solar-pv-profile,
-output_parameters,dict,n/a,{},,
\ No newline at end of file
+carrier,str,n/a,solar,,Energy carrier
+tech,str,n/a,pv,,Technology
+bus,str,n/a,,-el-bus,The bus this component is connected to
+capacity,,,,,Installed capacity
+capacity_cost,,,,,Annualized costs of capacity expansion
```

Comment:
Can we specify a unit here? It is 'MW' and 'Eur/MW' respectively according to `EnergyConversion_Capacity_Electricity_Solar_PV` and
`EnergyConversion_Capex_Electricity_Solar_PV` in Scalars.csv v0.02 and v0.03 or will it change?

Reply:
Thanks for finding this. This is Eur/MW.